src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+    scoreElem.appendChild(brds.scoreBrd.tblElem);
+
+    bigBoard = new UltimateBoard("brd-0", boardSize, brds.mainBrd.tblElem, brds.mainBrd.cellsArr);
+    scoreBrd = new Board("scrBrd", boardSize, brds.scoreBrd.tblElem, brds.scoreBrd.cellsArr);
+
+    addClickEventCells();
+}
+
+function initPlayers(plyr1, plyr2){
+    humPlayer = new HumenPlayer("You", "X");
+    compPlayer = new ComputerPlayer("Comp", "O");
+}
+
+function initGameInfo(gameInfoElem){
+    // const elem =  document.createElement("p");
+    //.setAttribute();
+
+    $(`#${elementsID.humPlayersign}`).html(`<p>Player 1: ${humPlayer.sign}</p>`);
+    $(`#${elementsID.compPlayersign}`).html(`<p>Player 2: ${compPlayer.sign}</p>`);
+}
+
+function updateCurrPlayer(){
+    if (currPlayer === humPlayer){
+        currPlayer = compPlayer;
+    } else {
+        currPlayer = humPlayer;
+    }
+    $(`#${elementsID.gameStatus}`).text(`Turn of: ${currPlayer.name}`);
+
+    if (currPlayer instanceof ComputerPlayer){
+        compMove();
+    }
+}
+
+function addClickEventCells(){
+    bigBoard.boardElem.addEventListener("click", cellClick);
+}
+
+function removeEvents(){
+    bigBoard.boardElem.removeEventListener("click", cellClick);
+}
+
+// Handlinng cell click event
+function cellClick(event){
+    if (currPlayer instanceof ComputerPlayer){
+        alert("Not your turn!");
+        return;
+    }
+
+    const clsSt = event.target.closest('.innerCell');
+
+    // If its not an inner cell that clicked
+    if (clsSt === null){
+        return;
+    }
+
+    playMove(clsSt.id);
+}
+
+// Extracting from cellId its positions in tables
+function extractCellInfo(cellId){
+
+   return {cellId: cellId,
+            rowBig: cellId[0],
+            colBig: cellId[1],
+            rowPly: cellId[2],
+            colPly: cellId[3]};
+}
+
+// Handling a play move
+function playMove(selectedCellId){
+    const clkCellInfo = extractCellInfo(selectedCellId);
+    const currPlayBoard = bigBoard.getCellPlayBoard(clkCellInfo.rowBig, clkCellInfo.colBig);
+
+    // If the clicked cell is empty and can be chosen
+    if (currPlayBoard.isEmptyCell(clkCellInfo.rowPly, clkCellInfo.colPly)){
+        $(`#${clkCellInfo.cellId}`).text(currPlayer.sign);
+
+        updateWins(clkCellInfo, currPlayBoard);
+
+        if (bigBoard.winStatus.isWin){
+            endGame(true);
+        } else {
+            updateCurrPlayer();
+        }
+
+        return true;
+    }
+}
+
+/**
+ * Update local win and global win in case there is one
+ * @param {object} cellInfo detailed selected cell info
+ * @param {PlayBoard object} playBrd current play board
+ */
+function updateWins(cellInfo, playBrd){
+    // Checking for local win (in play board, inside main cell)
+    const cellWin = playBrd.setCellAndWin(cellInfo.rowPly, cellInfo.colPly, currPlayer.sign);
+
+    // If there is a local win- checking for game win
+    if (cellWin){
+        bigBoard.setCellAndWin(cellInfo.rowBig, cellInfo.colBig, currPlayer.sign);
```

Comment:
Why do you need to set cell in "bigBoard" when you have the score board?

Also OOP design note for future with NodeJS: there won't be a score board in games running on server, it's for presentation only. Hmm..